spec/lib/vre/ch31_form_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'vre/ch31_form'
+
+RSpec.describe VRE::Ch31Form do
+  let(:claim) { create(:veteran_readiness_employment_claim) }
+  let(:user) { FactoryBot.create(:evss_user, :loa3) }
+  let(:service) { VRE::Ch31Form.new(user, claim) }
+  let(:new_address_hash) do
+    {
+      newAddress: {
+        isForeign: false,
+        isMilitary: nil,
+        countryName: 'USA',
+        addressLine1: '1019 Robin Cir',
+        addressLine2: nil,
+        addressLine3: nil,
+        city: 'Arroyo Grande',
+        province: 'CA',
+        internationalPostalCode: '93420'
+      }
+    }
+  end
+
+  describe '#submit' do
+    describe '#send_to_vre' do
```

Comment:
These are distinct methods, and only `#submit` is a valid method of the subject, `VRE::Ch31Form`. You may want to test that the `#submit` method calls `#send_to_vre` appropriately, but I'd move the expectations for the service class and methods into its own spec `vre_service_spec.rb`